testapi.pm

```diff
@@ -1026,6 +1021,9 @@ Run C<$command> via C<script_sudo> and then check by C<wait_serial> if its exit
 status is not zero.
 See C<wait_serial> for default timeout.
 
+For this to work the C<str> needs to agree with the corresponding C<str> marker
```

Comment:
What's `str` in this context? If you're referring to the internal variable I wouldn't mention it here since it's impossible for the caller to set it.

Reply:
`$str` is the hashed_string. i documented because it is easy to miss the fact that this part is associated with the hashed string on the other side ($distri->script_sudo).

Reply:
i can do so. I just thought to make it more official.

Reply:
Done